src/MOI_wrapper.jl

```diff
@@ -7,7 +7,7 @@ import Symbolics
 
 @enum(VariableType, INPUT, STATE, MODE)
 
-@enum(TimeType, UNKNOWN, CONTINUOUS, DISCRETE)
+@enum(TimeType, UNKNOWN, CONTINUOUS, DISCRETE, HYBRID)
```

Comment:
I think `HYBRID` falls into `CONTINUOUS` because the dynamic at the modes is continuous and the dynamic of the reset maps is discrete. Not sure whether we need a new enum state here.

Reply:
It was to be able to set hybrid system internally and triggered some actions like returning `HybridSystem` that Dionysos needs instead of `MathematicalSystems.ConstrainedBlackBoxControlDiscreteSystem` or `MathematicalSystems.ConstrainedBlackBoxControlContinuousSystem`. 

@blegat  But do we agree that when ever `add_transition` it's a hybrid system?

Reply:
Yes, we agree on that. On second thought, you're right, we need this